Prose/ProseLib/Sources/SidebarFeature/Footer/Footer.swift

```diff
@@ -28,7 +39,7 @@ struct Footer: View {
         // User avatar
         FooterAvatar(store: self.store.scope(state: \.avatar, action: Action.avatar))
 
-        WithViewStore(self.store, removeDuplicates: ==) { viewStore in
+        WithViewStore(self.store) { viewStore in
```

Comment:
Why not pull these properties below into the `ViewState` as well?

Reply:
For no reason, just forgot as it was not on my "bad performance" path. (And for this subview size, we don't usually create a `ViewState`)